packages/environment-check/test/checks/service-checks.test.ts

```diff
@@ -421,4 +421,31 @@ describe('Test service check functions', () => {
         expect(transportRequestResult.messages[0].severity).toBe(Severity.Error);
         expect(transportRequestResult.messages[1].severity).toBe(Severity.Debug);
     });
+
+    test('checkTransportRequests 403', async () => {
+        const getAdtService = jest.fn();
+        const axiosError = (status: 403) => {
+            return {
+                isAxiosError: true,
+                response: { status }
+            };
+        };
+
+        getAdtService.mockRejectedValueOnce(axiosError(403));
+
+        const abapServiceProvider = {
+            getAdtService: getAdtService
+        } as unknown as AbapServiceProvider;
+
+        // Test execution
+        const transportRequestResult = await checkTransportRequests(abapServiceProvider);
+
+        // Result check
+        expect(transportRequestResult.isTransportRequests).toBe(false);
+        expect(transportRequestResult.messages.length).toBe(3);
+        expect(transportRequestResult.messages[1].severity).toBe(Severity.Warning);
+        expect(transportRequestResult.messages[1].text).toBe(
+            'Guided Answers troubleshooting guide available at https://ga.support.sap.com/dtp/viewer/index.html#/tree/3046/actions/45995:45996:50742:46000:57266'
```

Comment:
I'd suggest you read the value from `guidedAnswersLink` rather than hard coding it.

Reply:
Ah yes, thank you. It's been updated.